agent/src/api/v1/jobs.rs

```diff
@@ -77,9 +77,9 @@ async fn run_job(
     // What we'll do later is accept this job for processing and send it to a thread or something.
     // But for now we do it right here, in our handler.
     // The correct response by design is a 202 Accepted plus the metadata object.
-    // TODO: SER-38 - capture exit code for failed jobs
     log::info!(
-        "about to run job name=TODO; id={}; executable size={}",
+        "about to run job name={}; id={}; executable size={}",
```

Comment:
hah, yesterday I looked at some logs and was like "I should find out why this job is called TODO" :D